recovery/util.h

```diff
@@ -16,6 +16,7 @@
 
 QByteArray getFileContents(const QString &filename);
 void putFileContents(const QString &filename, const QByteArray &data);
+bool backup(QString filename, QString ext="bak");
```

Comment:
Perhaps this should be `const QString &filename` just like `putFileContents` uses? ;-)